src/viam/sdk/common/utils.cpp

```diff
@@ -130,5 +130,18 @@ bool operator==(const response_metadata& lhs, const response_metadata& rhs) {
     return lhs.captured_at == rhs.captured_at;
 }
 
+bool from_dm_from_extra(const AttributeMap& extra) {
+    if (!extra) {
+        return false;
+    }
+    if (auto pos = extra->find("fromDataManagement"); pos != extra->end()) {
+        ProtoType value = *(pos->second);
+        const bool* boolValue = value.get<bool>();
+        return boolValue ? *boolValue : false;
+    } else {
+        return false;
```

Comment:
This function has three explicit `return false` statements, but I think it only needs one:

```
if (extra) {
    if (auto pos ...) {
        ...
        if (boolValue) { return *boolValue; }
    }
}
return false;
```

Reply:
changed the structure a little, but I do prefer using early returns to reduce nesting and make it easier to read